lib/ui/tabs/discover_tab.dart

```diff
@@ -46,6 +47,12 @@ class DiscoverTab extends StatelessWidget {
           }
         });
       },
+      loadingBuilder: (context) {
+        return SpinKitFadingCube(
+          color: Color(0xffB54646),
```

Comment:
You should never hard code color values like this, but instead, create a separate file to handle all these color constants with proper names denoting where they are used, or set colors of your theme in the MaterialApp widget.

For now, a simple fix is to replace all these `Color(0xffB54646)` with `Theme.of(context).primaryColor`.
Please make these changes everywhere you have hardcoded the color value.

Reply:
Sure. Guess this repo also needs another PR for cleaning the entire code and creating separate style files.

Reply:
Yes, you can make another PR for this if you like

Reply:
Lets leave it for someone else :)